tests/py_test.py

```diff
@@ -1367,5 +1399,22 @@ def main():
         buf, off = make_monster_from_generated_code()
         BenchmarkMakeMonsterFromGeneratedCode(bench_build, len(buf))
 
+    # run grpc tests
+    is_grpc = False
+    if is_grpc:
+        builder = flatbuffers.Builder(0)
+        name = builder.CreateString('Fred')
+        t = threading.Thread(target=run_server)
+        t.daemon = True
+        t.start()
+        server_instance_cv.acquire()
+        while server == 0:
+            server_instance_cv.wait()
+        server_instance_cv.release()
+        run_client(name)
```

Comment:
`name` is an offset to inside an unfinished FlatBuffer.. what is that supposed to do?